src/Components/Auth/ForgotPassword.js

```diff
@@ -0,0 +1,226 @@
+import React, { useState } from "react";
+import { Auth } from "aws-amplify";
+import { useHistory, Redirect } from "react-router-dom";
+import {Button,Form,FormGroup,Label,Input,FormText,Toast,ToastBody,ToastHeader,Modal,ModalBody,ModalHeader,Spinner} from "reactstrap";
+import { toast } from "react-toastify";
+import awsExports from "../../aws-exports";
+
+const ForgotPassword = (props) => {
+  const [isvalid, setIsValid] = useState(true);
+  const [mailSent, setMailSent] = useState(false);
+  const [pwdUpdated, setpwdUpdate] = useState(false)
+  const [loggingIn, toggleLoggingIn] = useState(false);
+  const toggle=()=>toggleLoggingIn(!loggingIn)
+  const [values, setValues] = useState({
+    user_email:"",
+    code: "",
+    confirm_password: "",
+    password: "",
+  });
+  const [visibility, setVisibility] = useState({
+    password:false,
+    confirm_password:false
+  });
+  const toggleVisibility = (field) =>
+    setVisibility({ ...visibility, [field]: !visibility[field] });
+
+  const history = useHistory();
+  const [errors, setError] = useState({});
+  const handleChange = (event) => {
+    const { name, value } = event.target;
+    setValues({
+      ...values,
+      [name]: value,
+    });
+  };
+
+  const sendMail = () => {
+    const user = values.user_email
+    toggle()
+    Auth.forgotPassword(user)
+      .then((data) => {
+        toast.info("Mail with verification code sent");
+        setMailSent(true)
+      })
+      .catch((err) => {
+        if(err.code==="UserNotFoundException") toast.error("User with given email address does not exist")
+        else toast.error(err.message)
```

Comment:
![image](https://user-images.githubusercontent.com/39180629/121424614-27319800-c98f-11eb-82ed-2ec6c1b6ed43.png)
Why is it showing `Username ...` instead of `Email ...` as you are taking email as input, right?